SQL/updates/60-61.py

```diff
@@ -0,0 +1,99 @@
+# :wave: hello fellow contributors, this script is brought to you ad-free by AffectedArc07 and Warriorstar
+# In order to run this script on Windows, you need to make sure you have Python **3** installed. Tested on 3.10.4
+# In addition you must have the mysql-connector-python module installed (can be done through pip :D)
+# if you do not have that module installed, you cannot run this script
+
+# To run this, supply the following args in a command shell
+# python 60-61.py address username password database
+# Example:
+# python 60-61.py 127.0.0.1 sirryan2002 myubersecretdbpassword paradise_gamedb
+
+from pathlib import Path
+from datetime import datetime
+import argparse
+import json
+
+import mysql.connector
+
+
+def log(msg):
+    print(f"[{datetime.utcnow()}] {msg}")
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "address", help="MySQL server address (use localhost for the current computer)"
+    )
+    parser.add_argument("username", help="MySQL login username")
+    parser.add_argument("password", help="MySQL login password")
+    parser.add_argument("database", help="Database name")
+
+    args = parser.parse_args()
+    db = mysql.connector.connect(
+        host=args.address, user=args.username, passwd=args.password, db=args.database
+    )
+    cursor = db.cursor()
+    log(f"Connected to {args.database}")
+
+    # want these ordered by length from longest to shortest so shorter replacements
+    # don't replace pieces of larger replacements
+    renames = sorted(
+        json.load(
+            open(Path(__file__).parent / "migration_data/snake_case_type_remap.json")
```

Comment:
I assume this means the script will run from anywhere, like if the CWD is the repo root, it will load the data properly?

Reply:
Yup:
```sh
$ echo "print(__file__)" >> tools/test.py
$ python tools/test.py
D:\ExternalRepos\third_party\Paradise\tools\test.py
```